torch/_export/serde/serialize.py

```diff
@@ -1850,6 +1853,11 @@ def deserialize(
                 allow_non_fake_inputs=True,
                 shape_env=self.shape_env,
             )
+            self.sympy_functions = {
+                "ToFloat": torch.utils._sympy.functions.ToFloat,
+                "TruncToInt": torch.utils._sympy.functions.TruncToInt,
+                # add more sympy functions here
+            }
```

Comment:
question, not comment: we need to add all functions here right? Not just the unsupported ones

Reply:
Yeah, but I don't know which ones should be here vs. handled elsewhere; some of them are int operations, like Mod and FloorDiv, should they be here? Or is it just the floor <-> int ones?

If you're going to do a pass anyway, maybe you can fill this up? Or let me know what else to add here.

Reply:
Added all.